yogstation/code/modules/language/russian.dm

```diff
@@ -0,0 +1,14 @@
+/datum/language/russian
+	name = "Russian"
+	desc = "The common language of Russian citizens. Still exists among Russian metro dwellers.."
+	speech_verb = "govorit"
+	ask_verb = "prosit"
+	exclaim_verb = "vosklitsat"
+	key = "r"
+	space_chance = 40
+	syllables = list(
+		"da", "dobraye", "ootro", "dobriy den'", "spaseeba", "eezveeneete"
```

Comment:
You could ask L0rd_English or one of the other ruskies to help you get more syllables for this list.

Reply:
I would put a thumbs up emoji here, but it seems that I cannot. I will go do that, this would be better conveyed with a thumbs up emoji though.

Reply:
Removed because stupid, fight me in the ring